os/tools/mkbootparam.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+############################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+import os
+import sys
+import struct
+
+config_file_path = os.path.dirname(__file__) + '/../.config'
+output_path = os.path.dirname(__file__) + '/../../build/output/bin/'
+file_path = output_path + 'bootparam.bin'
+
+SIZE_OF_BOOTPARAM_PART = 8192
+
+###########################################################################################
+#
+# This script generates Boot Parameters for kernel.
+#
+# Boot Parameter information :
+# +---------------------------------------------------------------------+
+# | Active Idx |  First Address  | Second Address  |      Reserved      |
+# |  (1 byte)  |    (4 bytes)    |    (4 bytes)    | (8Kbytes - 9bytes) |
+# +---------------------------------------------------------------------+
+#  * The Boot Parameters (8KB) should be downloaded and be located at the end of a flash.
+#
+###########################################################################################
+
+def get_config_value(file_name, config):
+    with open(file_name, 'r+') as fp:
+        lines = fp.readlines()
+        found = False
+        for line in lines:
+            if config in line:
+                value = (line.split("=")[1])
+                value = value.replace('"','').replace('\n','')
+                found = True
+                break
+        fp.close()
```

Comment:
In python, `with` will close the file automatically. So you don't nee to close manually.